CPPLINT.cfg

```diff
@@ -1,2 +1,2 @@
 set noparent
-filter=-runtime/reference,-runtime/string,-build/c++11,-build/include_subdir,-whitespace/forcolon,-build/namespaces_literals,-readability/check
+filter=-runtime/reference,-runtime/string,-build/c++11,-build/include_subdir,-whitespace/forcolon,-build/namespaces_literals,-readability/check,-build/include_what_you_use
```

Comment:
`build/include_what_you_use` produces many false positives: it doesn't check the namespace, just the name of the function. I ran into another example in #1648
```
szaszm@szaszm-pc:~/nifi-minifi-cpp-3/build-gcc-make$ make linter
Built target minifi-usb-camera-extensions-linter
/home/szaszm/nifi-minifi-cpp-3/libminifi/include/utils/detail/MonadicOperationWrappers.h:61:  Add #include <algorithm> for transform  [build/include_what_you_use] [4]
Done processing /home/szaszm/nifi-minifi-cpp-3/libminifi/include/utils/detail/MonadicOperationWrappers.h
/home/szaszm/nifi-minifi-cpp-3/libminifi/include/DiskSpaceWatchdog.h:33:  At least two spaces is best between code and comments  [whitespace/comments] [2]
/home/szaszm/nifi-minifi-cpp-3/libminifi/include/DiskSpaceWatchdog.h:52:  At least two spaces is best between code and comments  [whitespace/comments] [2]
Done processing /home/szaszm/nifi-minifi-cpp-3/libminifi/include/DiskSpaceWatchdog.h
Total errors found: 1
/home/szaszm/nifi-minifi-cpp-3/libminifi/include/properties/Decryptor.h:46:  Add #include <algorithm> for transform  [build/include_what_you_use] [4]
Done processing /home/szaszm/nifi-minifi-cpp-3/libminifi/include/properties/Decryptor.h
/home/szaszm/nifi-minifi-cpp-3/libminifi/include/core/logging/LoggerConfiguration.h:65:  Add #include <algorithm> for transform  [build/include_what_you_use] [4]
Done processing /home/szaszm/nifi-minifi-cpp-3/libminifi/include/core/logging/LoggerConfiguration.h
```